README.md

```diff
@@ -22,6 +22,17 @@ of the following ways:
 - Create a [Github issue](https://github.com/pulibrary/dls-handbook/issues)
 - Message us in #digital_library in PULibrary's Slack
 
+## Our Vision
+
+Princeton University Library has defined its [Mission, Vision, and North Star
+Statements](https://library.princeton.edu/about) and DLS has worked with its
+partners to define the Digital Library's part in the wider vision and how we can
+best move it forward. It is as follows:
+
+Princeton University's Digital Library will become an inspiring environment for global communities to engage with diverse collections. To support this goal, our Digital Library platform will enable easy ingest, long-term preservation, joyful discovery, and transformative use of digital materials.
+
+DLS will prioritize development and processes in such a way to have the most impact towards this vision, and we encourage our partners and stakeholders to imagine ways for us to get there together.
```

Comment:
I like this part. I think it should be "in such a way as to have." Maybe we should expand "development and processes" into, like, "development work and organization processes"

Reply:
I really like expanding it, @hackartisan